src/autoProcess.ts

```diff
@@ -270,6 +272,15 @@ export function autoPreprocess(
         map = transformed.map;
       }
 
+      const transformed = await runTransformer('globalRule', null, {
```

Comment:
Hmmm, couldn't this thrown an error if `postcss` is not installed? We need some kind of check to run this transformer only if `postcss` is installed